WalletWasabi/Blockchain/TransactionOutputs/CoinsRegistry.cs

```diff
@@ -110,10 +117,7 @@ public ICoinsView Remove(SmartCoin coin)
 				{
 					if (!Coins.Remove(toRemove))
 					{
-						if (SpentCoins.Remove(toRemove))
-						{
-							// Clusters.Remove(toRemove);
-						}
+						SpentCoins.Remove(toRemove);
```

Comment:
For the record for others who are reviewing, this has nothing to do with this PR. It just says we don't undo the cluster because the relationship is already exposed.